lib/cell.rb

```diff
@@ -0,0 +1,21 @@
+class Cell
+attr_reader :coordinate,
+            :ship
+
+  def initialize(coordinate)
+    @coordinate = coordinate
+    @ship = nil
+  end
+
+  def empty?
+    ship.nil?
+  end
+
+  def place_ship(new_ship)
+    @ship = new_ship
+  end
+
+  def fired_upon?
+    require 'pry'; binding.pry
```

Comment:
got stuck here. check out methods from ship class, maybe create some instance variables to work with? shot, miss, hit?